We need to ensure that off64_t is defined when building with
_LARGEFILE64_SOURCE.

--- a/zconf.h
+++ b/zconf.h
@@ -379,6 +379,10 @@
 #  include <sys/types.h>
 #endif
 
+#ifdef _LARGEFILE64_SOURCE
+#include <sys/types.h>
+#endif
+
 #ifndef SEEK_SET
 #  define SEEK_SET        0       /* Seek from beginning of file.  */
 #  define SEEK_CUR        1       /* Seek from current position.  */
